benchexec/tablegenerator/react-table/src/components/SelectColumn.js

```diff
@@ -52,10 +53,39 @@ export default class SelectColumn extends React.Component {
   };
 
   renderColumns = index => {
-    return this.state.list[index].columns.map(column => (
+    this.listTable = [];
+    let idxColumn = [];
+    this.state.list.forEach(tool => {
+      this.selectable.forEach(headerRow => {
+        let length = tool.columns.length;
+        tool.columns.forEach(function(column, idx, i) {
```

Comment:
This loops over `tool.columns`, but only does something in one loop iteration (either where the matching column is or at the end). All other loop iterations are empty, so this seems like an unnecessary loop. Wouldn't it be easier to just check if `tool.columns` contains a matching `column` with an `if`?

Reply:
Done